server/routes/index.ts

```diff
@@ -243,7 +248,7 @@ export function defineRoutes(router: IRouter, esClient: IClusterClient) {
         body: schema.any(),
       },
     },
-    async (context, request, response) => {
+    async (context, request, response): Promise<IKibanaResponse<any | ResponseError>> => {
```

Comment:
We can also have the `async (context, request, response): Promise<IKibanaResponse<any | ResponseError>>` for other async function in this file.

Reply:
agreed, updated. We can define specific type later